src/components/DraggableCard/DraggableCard.js

```diff
@@ -36,8 +37,64 @@ export default function DraggableCard({
   fontSize,
   id,
   showRawContent,
+  parentRef,
 }) {
   const classes = useStyles()
+  const cardRef = useRef(null)
```

Comment:
Should we create a custom hook with this new code that was added for updating the bounds? could be named useBoundsUpdater or useUpdateBounds